tutorials/Creating a new Lumigator endpoint.md

```diff
@@ -0,0 +1,581 @@
+# Creating a new Lumigator endpoint
+
+This document shows how current endpoints in lumigator work and describes the process needed to create a new one. This
+might or might not become part of the public documentation - I think it might be useful for users to expand the tool
```

Comment:
we can remove this `This
might or might not become part of the public documentation - I think it might be useful for users to expand the tool`